pages/common/cut.md

```diff
@@ -12,7 +12,7 @@
 
 - Cut out everything from the 3rd character to the end of each line:
 
-`cut -c{{3-}}`
+`cut -c {{3-}}`
```

Comment:
This was valid syntax before. The space here is completely optional actually. Personally, I'd like to leave this as-is - as some commands (e.g. 7-zip) require the lack of space, and that can be confusing if it's a one-off thing.

Reply:
I think it is more to do with consistency. If you see the above 2 examples, they also use `-c` but have a space. I do not mind either way, but if we are not using space, then I would change the above examples too.

Reply:
Ah. Let's change this then. I've just seen a number of PRs making this kind of change recently.

Reply:
Yes, exactly as @agnivade puts it, this change is for the sake of consistency. Either we omit the space or insert it, either one of those but preferably the same in all examples.

Sorry for replying late to this, luckily @agnivade motivated the change for me, thanks!